pages/careers/_open-positions.html

```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
+    <p>We give software teams a complete, world-class cloud infrastructure that lets them go to prod in days. Typically,
+       the process of launching prod infrastructure in the cloud is awful. It takes months of tedious work, hundreds of
+       thousands of dollars, and often results in poorly maintained code and anti-patterns in the end. We call our company
+       "Gruntwork" because we take away this Gruntwork for our clients so that they can focus on the parts of their software
+       that are unique to them.</p>
+    <p>We are 100% remote, ~15 people, profitable, growing (even during COVID!) and fully bootstrapped with no debt and
+       no investors. Our small company works with both startups and name companies like Toyota, Adobe, and the US House
+      of Representatives. We also <a href="#gruntwork-is-a-human-friendly-company">care a lot about humans.</a></p>
+
+    <h4>What's the opportunity?</h4>
+    <ul>
+      <li>
+        <strong>Product Design.</strong> We're building a SaaS platform that will become the primary customer experience of our product. Today, customers access our product as private GitHub repos plus a patchwork of links to training and documentation. Working with you, we envision a portal where customers login to discover and use all the elements of our existing product, brand it to themselves, and ultimately offer a UI-based way to learn, deploy and manage infrastructure in the cloud, all backed by infrastructure as code under the hood.
+      </li>
+      <li>
+        <strong>Branding and Marketing Website.</strong> We have an improved understanding of our business that we'd like to reflect in a new public marketing website. We'll look to you lead the design effort here.
+      </li>
+      <li>
+        <strong>Design system.</strong> You'll be our first full-time designer so you will set the design direction for the entire company across all areas of design, working directly with the founders and closely with engineers.
+      </li>
+    </ul>
+
+    <p>Although you'll be the only professional designer on the team, our company cares deeply about design so you'll have many eager collaborators when it comes time to brainstorm.</p>
+
+    <h4>What skills are we looking for?</h4>
+
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong> When we present our product vision, we'll look to you to offer opinions, insights, and methodologies to help us translate that vision into a fantastic product.
+      </li>
+      <li>
+        <strong>Design skill.</strong> At the end of the day, the designs you create should look great and be easy to use. Our hope is that you can make our users smile.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong> We don't expect you to have a complete vision for our product on day 1, but we're looking for someone who either already understands our domain (DevOps, CI/CD, Infrastructure as Code, AWS), or who feels comfortable ramping up on it.
```

Comment:
I think I'd rephrase this as, "We don't expect you to have a complete vision for our product on day 1, but to be able to design products in our space, you must be able to thoroughly ramp up on our domain (DevOps, software delivery, cloud software, CI / CD, infrastructure as code). Prior experience software engineering, working on dev tools, and other highly technical domains is a big plus."

Reply:
Added in 629e840.